apps/system/js/accessibility.js

```diff
@@ -284,6 +284,22 @@
     },
 
     /**
+     * Start a timeout that waits to display hints
+     * @memberof Accessibility.prototype
+     */
+     setHintsTimeout: function ar_setHintsTimeout(hints) {
+      var accessibilityObj = this;
+      this.hintTimer = setTimeout(function() {
```

Comment:
Name this function (```function onHintTimeout() {...}```) . Things like this are useful for debugging because you would get a name of the function in the callstack if you get an exception, for example, instead of 'undefined'